README.md

```diff
@@ -1,5 +1,18 @@
-# HelloWebView - start
+# WebView2 Samples
+
+This repository contain examples of how to use the [WebView 2 API](https://docs.microsoft.com/en-us/microsoft-edge/hosting/webview2). 
+
+## 1. Getting Started Guide
+
+Start with the [Microsoft Edge WebView2 getting-started guide](https://docs.microsoft.com/en-us/microsoft-edge/hosting/webview2/gettingstarted) to learn how to setup a WebView within a Win32 application.
+
+This Sample is found within the GettingStartedGuide directory.
+
+## 2. WebView2 Browser Example
+
+Follow the [WebView2Browser guide](https://github.com/MicrosoftEdge/WebView2Browser) to learn how to build an application that utilizes multiple WebViews.
+
+Clone the project by running `git clone https://github.com/MicrosoftEdge/WebView2Browser.git` to get started.
```

Comment:
I don't think this line is necessary.  A link to the WebView2Browser repo should be enough.